lib/datasource/index.ts

```diff
@@ -87,7 +205,6 @@ function getRawReleases(config: PkgReleaseConfig): Promise<ReleaseResult> {
   }
   return global.repoCache[cacheKey];
 }
-
```

Comment:
Why?

Reply:
Pre-existing code which I haven't touched I believe.

Reply:
Maybe your editor has removed the empty lines, please try to revert this.